docs/configuration/02-keybindings.md

```diff
@@ -79,6 +79,12 @@ lvim.builtin.which_key.mappings["P"] = {
   "<cmd>lua require'telescope'.extensions.project.project{}<CR>", "Projects" 
 }
 ```
+Adding a key to a submenu. 
+``` lua
+lvim.builtin.which_key.mappings["TP"] = { 
```

Comment:
can you add a sample that currently exists like `lt` or `gd` or sth 🥺

Reply:
Yes I could i just thought it would be confusing because it is how to add a new keybinding. People might think that it is how to overwrite or smth or be confused. But I can change if you want

Reply:
no what i mean is that, add a key to a **submenu that already exists**

Reply:
I kept the P as it was the same as the other example, but changed it to tP for telescope => project